plugins/lookup/onepassword.py

```diff
@@ -551,7 +580,7 @@ def _get_cli_class(self):
         for cls in OnePassCLIBase.__subclasses__():
             if cls.supports_version == version.split(".")[0]:
                 try:
-                    return cls(self.subdomain, self.domain, self.username, self.secret_key, self.master_password)
+                    return cls(self.subdomain, self.domain, self.username, self.secret_key, self.master_password, self.service_account_token)
```

Comment:
The v1 CLI class does ignores this value. Please change `OnePassCLIv1.full_signin` to fail in case `service_account_token is not None` so that the user knows the value doesn't work.

Reply:
I added a check in the v1 CLI class. Please check if this is fine for you.